src/components/Editor/DebugLine.js

```diff
@@ -8,43 +8,44 @@ import { markText, toEditorPosition } from "../../utils/editor";
 import { getDocument } from "../../utils/editor/source-documents";
 
 import { connect } from "react-redux";
-import { getVisibleSelectedFrame, getPause } from "../../selectors";
+import {
+  getVisibleSelectedFrame,
+  getPause,
+  getSelectedLocation
+} from "../../selectors";
+
+import type { Frame, Location, Pause } from "debugger-html";
 
 type Props = {
   editor: Object,
-  selectedFrame: Object,
-  pauseInfo: Object
-};
-
-type State = {
-  debugExpression: {
-    clear: Function
-  }
+  selectedFrame: Frame,
+  selectedLocation: Location,
+  pauseInfo: Pause
 };
 
 type TextClasses = {
   markTextClass: string,
   lineClass: string
 };
-export class DebugLine extends Component<Props, State> {
-  constructor() {
-    super();
-    this.state = { debugExpression: { clear: () => {} } };
-  }
+
+export class DebugLine extends Component<Props> {
+  debugExpression: null;
```

Comment:
I'm switching `debugExpression` to a property so that setting it does not trigger an update.